app/styles/layout.scss

```diff
@@ -0,0 +1,125 @@
+.fullpage {
+  display: flex;
+  position: relative;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  margin: 0 auto;
+}
+
+.navside {
+  display: flex;
+  position: fixed;
```

Comment:
You can remove `position: fixed` and this will help to make the sidebars perfectly take their room on each side.